server/scripts/vet-api-check.sh

```diff
@@ -55,6 +55,7 @@ OUTPUT_EXCLUDING_IGNORED=$(echo "$OUTPUT" | grep -Fv \
     -e 'Cannot find /api/v4/users/login/desktop_token method: POST in OpenAPI 3 spec.' \
     -e 'Cannot find /api/v4/users/notify-admin method: POST in OpenAPI 3 spec.' \
     -e 'Cannot find /api/v4/users/trigger-notify-admin-posts method: POST in OpenAPI 3 spec.' \
+    -e 'Cannot find /api/v4/reports/users method: GET in OpenAPI 3 spec.' \
```

Comment:
Why is this needed? Somehow feels the wrong place to add this, but I don't understand 100% what is going on on this file :sweat_smile:

Reply:
Its mentioned in the PR where this was added - https://github.com/mattermost/mattermost/pull/23974

Pasting here for reference -

```
The underlying mattermost-govet tool effectively hasn't been called for some time, as we weren't checking out and building the spec files to pass. Now that the API is in the monorepo, build it locally and check against it.

Unfortunately, our API documentation isn't up-to-date, and this PR isn't fixing that. For now, add a discrete make vet-api and workflow that won't block the build until the API documentation is back in sync and can be merged into the existing make vet directive. Additionally, suppress errors from "known mismatches" (while still logging them), so we can at least start checking the API moving forward.
```

Reply:
@harshilsharma63 This is only for existing API stuff that was added before the check was restored. This API actually is documented [here](https://github.com/mattermost/mattermost/blob/3e16d621a09e63c9611508ae91d79665e3568ba2/api/v4/source/users.yaml#L3254) so this needs to be moved over and this check needs to be enabled.

Reply:
...which it looks like you did. So I don't know why this comment is needed.

Reply:
Fixed! So I had added a new `report.yaml` file but the code that consumes those yaml files needs to be told the names of all files, and doesn't automatically consume all yaml files in the directory 🤦‍♂️

Reply:
So telling it the new file's name fixed it.